auth/user_mgt.go

```diff
@@ -62,20 +63,33 @@ type UserInfo struct {
 
 // multiFactorInfoResponse describes the `mfaInfo` of the user record API response
 type multiFactorInfoResponse struct {
-	MFAEnrollmentID string `json:"mfaEnrollmentId,omitempty"`
-	DisplayName     string `json:"displayName,omitempty"`
-	PhoneInfo       string `json:"phoneInfo,omitempty"`
-	EnrolledAt      string `json:"enrolledAt,omitempty"`
+	MFAEnrollmentID string    `json:"mfaEnrollmentId,omitempty"`
+	DisplayName     string    `json:"displayName,omitempty"`
+	PhoneInfo       string    `json:"phoneInfo,omitempty"`
+	TOTPInfo        *TOTPInfo `json:"totpInfo,omitempty"`
+	EnrolledAt      string    `json:"enrolledAt,omitempty"`
+}
+
+// TOTPInfo describes a user enrolled second totp factor.
+type TOTPInfo struct{}
+
+// PhoneMultiFactorInfo describes a user enrolled second phone factor.
+type PhoneMultiFactorInfo struct {
+	PhoneNumber string
+}
+
+type multiFactorEnrollments struct {
+	Enrollments []*multiFactorInfoResponse `json:"enrollments"`
 }
 
 // MultiFactorInfo describes a user enrolled second phone factor.
-// TODO : convert PhoneNumber to PhoneMultiFactorInfo struct
 type MultiFactorInfo struct {
-	UID                 string
-	DisplayName         string
-	EnrollmentTimestamp int64
-	FactorID            string
-	PhoneNumber         string
+	UID                  string
+	DisplayName          string
+	EnrollmentTimestamp  int64
+	FactorID             string
+	PhoneMultiFactorInfo *PhoneMultiFactorInfo
+	TOTPMultiFactorInfo  *TOTPInfo
```

Comment:
Change to TOTPMultiFactorInfo to consist with PhoneMultiFactorInfo?

Reply:
It is called `TOTPInfo` in `authFactorInfo` which is what the server definition looks like. For consistency alone, do you think it is worth creating another empty struct?

Reply:
Yeah, I think so. Including "MultiFactor" is necessary to me.